From b104124c6232d18ae72dfde33fdaa6fd05e91b8b Mon Sep 17 00:00:00 2001
From: Casey Schaufler <casey@schaufler-ca.com>
Date: Thu, 19 Dec 2013 16:49:28 -0800
Subject: [PATCH 12/24] Run systemd-journald with the hat ("^") Smack label.

The journal daemon needs global read access to gather information
about the services spawned by systemd. The hat label is intended
for this purpose. The journal daemon is the only part of the
System domain that needs read access to the User domain. Giving
the journal daemon the hat label means that we can remove the
System domain's read access to the User domain.

Change-Id: Ic22633f0c9d99c04f873be8a346786ea577d0370
Signed-off-by: Casey Schaufler <casey.schaufler@intel.com>
---
 units/systemd-journald.service.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/units/systemd-journald.service.in b/units/systemd-journald.service.in
index 7013979..532e0ec 100644
--- a/units/systemd-journald.service.in
+++ b/units/systemd-journald.service.in
@@ -20,6 +20,7 @@ Restart=always
 RestartSec=0
 NotifyAccess=all
 StandardOutput=null
+SmackExecLabel=^
 CapabilityBoundingSet=CAP_SYS_ADMIN CAP_DAC_OVERRIDE CAP_SYS_PTRACE CAP_SYSLOG CAP_AUDIT_CONTROL CAP_CHOWN CAP_DAC_READ_SEARCH CAP_FOWNER CAP_SETUID CAP_SETGID
 WatchdogSec=1min
 
-- 
1.8.4.5

